src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService usersData = new UserService();
+
     public boolean login(String email, String password) {
+        for (User user: usersData.getUsers()) {
+            if (user.getEmail().equals(email) && user.getPassword().equals(password)) {
+                return true;
+            }
+        }
```

Comment:
Let's use `findByEmail()` method from `userService` to not duplicate our logic and here we can just do the null check and password check
```suggestion
```